### ISSUE TITLE
Boxed supposed "bug"

### ISSUE THREAD

**Issue by qarbon1** [BentoBoxWorld/Boxed#81]
### Expected behavior

No errors

### Observed/Actual behavior

Errors + server overloaded

### Steps/models to reproduce

https://pastebin.com/n8rrLFdt
https://pastebin.com/x5afUJic

### BentoBox version

![2024-07-20_13 04 32](https://github.com/user-attachments/assets/fcf4f362-8bfe-461a-90dc-2ee9399a6469)

### Plugin list

BentoBox, BentoBox-Bank, BentoBox-Boxed, BentoBox-Challenges, BentoBox-Likes, DeluxeMentions, DeluxeMenus, Essentials, EssentialsSpawn, FastAsyncWorldEdit,
LPC, LuckPerms, Multiverse-Core, PlaceholderAPI, PluginPortal, PlugManX, spark, TAB, Vault, WorldGuard

**Comment by qarbon1** [comment]
Just switched thru my spawn world and box world..

**Comment by tastybento** [comment]
Hmm, both of the logs show lag. The first is with InvSwitcher and the second is due to chunks being processed when the player entered the world. Both of these are lagging within the server itself and I don't currently understand why they would do that. It may be that 1.21 is less able to handle this stuff. I'll try and see if there's anything I can do here...

**Comment by qarbon1** [comment]
Should I maybe prerender the chunks?

**Comment by tastybento** [comment]
Yes, you could.

**Comment by qarbon1** [comment]
Could it be a solution that I added more RAM to the server itself?

**Comment by tastybento** [comment]
Yes, it should work. Make sure you assign the RAM in the java command, e.g., -Xms12G -Xmx12G for 12 Gig. See the readme for the full command.

**Comment by qarbon1** [comment]
I am current using aikar's flags.

**Comment by qarbon1** [comment]
https://pastebin.com/U1ZU2eWm
